DOCS-791: Update Rover tutorial with C++ (#2580)
viamrobotics/docs: docs/tutorials/get-started/try-viam-sdk.md

## Install a Viam SDK

Install one of the following SDKs on your local computer:

- the [Viam Python SDK](https://python.viam.dev/)
- the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme)
- the [Viam Flutter SDK](https://flutter.viam.dev/)

> - the [TypeScript SDK](https://ts.viam.dev/)

Reviewer: ```suggestion
- the [Viam TypeScript SDK](https://ts.viam.dev/)
```
(opt) just for consistency

> - the [C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk/#readme)

Reviewer: ```suggestion
- the [Viam C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk/#readme)
```
(opt) just for consistency

{{<video webm_src="/tutorials/try-viam-sdk/square-test-rover.webm" mp4_src="/tutorials/try-viam-sdk/square-test-rover.mp4" alt="An example flutter app moving a Try Viam rental rover in a square" poster="/tutorials/try-viam-sdk/square-test-rover.jpg">}}

{{% /tab %}}
{{% tab name="C++" %}}

The first thing you need to do is import the [base component](https://cpp.viam.dev/classviam_1_1sdk_1_1Base.html). The base is responsible for controlling the motors attached to the base of the rover. Add the following line of code to your imports:

```cpp {class="line-numbers linkable-line-numbers"}
#include <viam/sdk/components/base/client.hpp>
```

Additionally, add the following namespaces under your imports:

```cpp {class="line-numbers linkable-line-numbers"}
using namespace viam::sdk;
using std::cerr;
using std::cout;
using std::endl;
```

Next, you need to initialize your rover base.

> In the main function, after you connect, paste the code from lines 19-22. On the Try Viam rental rovers, the default base name is `viam_base`. If you have a different base name, update the name in your code.

Reviewer: ```suggestion
In the main function, after you connect, paste the code from lines 21-30. On the Try Viam rental rovers, the default base name is `viam_base`. If you have a different base name, update the name in your code.
```
This doesn't match what's highlighted in sample. I think this would?
